lisa/util/__init__.py

```diff
@@ -750,3 +753,25 @@ def check_till_timeout(
         sleep(interval)
     if timer.elapsed() >= timeout:
         raise LisaException(f"timeout: {timeout_message}")
+
+
+def retry_with_excluded_exceptions(
+    tries: int,
+    delay: float,
+    exceptions_to_skip: Tuple[Type[Exception], ...] = (),
```

Comment:
Use List instead of Tuple. The values should use `[ ]`, not `( )`.

Reply:
name it like `skipped_exception`, so just one underscore.

Reply:
updated